kratos/elements/levelset_convection_element_simplex.h

```diff
@@ -351,9 +345,9 @@ class LevelSetConvectionElementSimplex
             {
                 h_inv += DN_DX(i,k)*DN_DX(i,k);
             }
-            h += 1.0/h_inv;
+            h = std::max(h, 1.0 / h_inv);
         }
-        h = sqrt(h)/static_cast<double>(TNumNodes);
```

Comment:
You are also modifying how the element size is calculated.

Reply:
std::sqrt if possible (c and std sqrt return the same value, but the proper thing is to use the std one)

Reply:
Wait, it is a removed line, sorry for the noise

Reply:
Yes, this is change @RiccardoRossi suggested us a long time ago, but we never pushed it back to the core. It's been working fine for us, but I guess it's open to debate.

Reply:
> Yes, this is a change @RiccardoRossi suggested us a long time ago, but we never pushed it back to the core. It's been working fine for us, but I guess it's open to debate.

OK. Just for me to know, can you briefly summarize the implications of using this new `h`?

Reply:
I do not really know the rationale behind this change (this was done before I entered in altair), but I guess the effect is that the computed element size is bigger and therefore so they are the stabilization terms. I am not sure which one is more correct (maybe @RiccardoRossi can bring some light here), but we should converge in one solution.